src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -141,7 +141,16 @@ private void CheckAbstractClassImplementations(DiagnosticBag diagnostics)
 
                     if ((object)synthesizedImplementation != null)
                     {
-                        synthesizedImplementations.Add(synthesizedImplementation);
+                        if (synthesizedImplementation.IsVararg)
+                        {
+                            diagnostics.Add(
+                                ErrorCode.ERR_InterfaceImplementedImplicitlyByVariadic, 
```

Comment:
>ErrorCode.ERR_InterfaceImplementedImplicitlyByVariadic,  [](start = 32, length = 56)

Can we report this error under umbrella of ReportImplicitImplementationMatchDiagnostics method?

Reply:
This is the place where we definitely know that we need a vararg bridge regardless of the reasons. 
I think placing check here is appropriate.

---
In reply to: [167038304](https://github.com/dotnet/roslyn/pull/24679#discussion_r167038304) [](ancestors = 167038304)